### PR TITLE
Add some more loot options

### PR DESCRIPTION
**Pull request by SoraTenshi**
Of course it's untested.

### CODE REVIEW

File: src/service/lootService.ts [hunk 1]
```diff
@@ -239,6 +243,42 @@ const lootTemplates: loot.LootTemplate[] = [
         emote: "🏆",
         asset: "assets/loot/17-achievement.png",
     },
+    {
+        id: LootTypeId.GME_AKTIE,
+        weight: 5,
+        displayName: "GME-Aktie",
+        titleText: "Eine GME Aktie",
+        description: "Der squeeze kommt bald!",
+        emote: "📈",
```
Comment (holzmaster) [inline src/service/lootService.ts:252]:
```suggestion
        emote: "📉",
```

File: src/service/lootService.ts [hunk 2]
```diff
@@ -239,6 +243,42 @@ const lootTemplates: loot.LootTemplate[] = [
...
+        asset: "assets/loot/18-gme.jpg",
+    },
+    {
+        id: LootTypeId.FERRIS,
+        weight: 3,
+        displayName: "Ferris",
+        titleText: "Ferris - Die Krabbe",
```
Comment (holzmaster) [inline src/service/lootService.ts:259]:
```suggestion
        titleText: "Einen Ferris - Die Krabbe",
```

File: src/service/lootService.ts (hunk 1 above)
Comment (SoraTenshi) [reply src/service/lootService.ts:252]: Hmm i had the idea for bitcoin and would much rather assign the emote to the bitcoin.

File: src/service/lootService.ts [hunk 3]
```diff
@@ -239,6 +243,43 @@ const lootTemplates: loot.LootTemplate[] = [
...
+        emote: "🍎",
+        asset: "assets/loot/20-homepod.jpg",
+    },
+    {
+        id: LootTypeId.RADIOACTIVE_WASTE,
+        weight: 1,
+        displayName: "Radioaktiver Müll",
+        titleText: "Radioaktiver Müll",
+        description:
+            "Sollte dir ja nichtsmehr anhaben, bist ja durch den Server schon genug verstrahlt!",
```
Comment (holzmaster) [inline src/service/lootService.ts:279]:
```suggestion
            "Sollte dir ja nichts mehr anhaben, du bist ja durch den Server schon genug verstrahlt 🤷‍♂️",
```

File: src/service/lootService.ts (hunk 1 above)
Comment (holzmaster) [reply src/service/lootService.ts]: `:peepoNotStonks:`?

Comment (SoraTenshi) [reply src/service/lootService.ts]: Ah they're not identifiers so fine by me.

File: src/service/lootService.ts [hunk 4]
```diff
@@ -239,6 +243,43 @@ const lootTemplates: loot.LootTemplate[] = [
         emote: "🏆",
         asset: "assets/loot/17-achievement.png",
     },
+    {
+        id: LootTypeId.GME_AKTIE,
+        weight: 5,
+        displayName: "GME-Aktie",
+        titleText: "Eine GME Aktie",
```
Comment (holzmaster) [inline src/service/lootService.ts:250]:
```suggestion
        titleText: "Eine wertlose GME-Aktie",
```

**Pull request merged**